dev-tools/scripts/addDepsToChanges.py

```diff
@@ -102,6 +104,11 @@ def update_changes(filename, version, changes_lines):
                         buffer.append(change_line)
                         buffer.append("\n")
                     continue
+            else:
```

Comment:
Don't think you need an `else` here, but it doesn't really matter.

Reply:
Good catch. Removed it, and also tightened the requiremnet of `---` line after version header.